fml/raster_thread_merger_unittests.cc

```diff
@@ -301,3 +304,93 @@ TEST(RasterThreadMerger, HandleTaskQueuesAreTheSame) {
   term1.Signal();
   thread1.join();
 }
+
+TEST(RasterThreadMerger, FirstTaskMergesAndUnmergesThreads) {
```

Comment:
nice!

Reply:
Separated this test into 2 tests since you reviewed last time.